src/Base/Admin/AdminServiceProvider.php

```diff
@@ -35,7 +35,10 @@ public function filterPostLink(string $link, \WP_Post $post, bool $leavename, $s
      */
     public function filterPreviewLink(string $link, \WP_Post $post): string
     {
-        if ($post->post_type !== 'pdc-item' || !$this->plugin->settings->isPortalSlugValid()) {
+        if (
+            ! in_array($post->post_type, ['pdc-item', 'pdc-category', 'pdc-subcategory']) ||
+            !$this->plugin->settings->isPortalSlugValid()
```

Comment:
`! $this->plugin->`, om het uniform te houden graag nog een spatie tussen `!` en `$this`.